packages/exchange/package.json

```diff
@@ -31,7 +31,7 @@
         "start-ganache": "ganache-cli -m 'chalk park staff buzz chair purchase wise oak receive avoid avoid home' -l 8000000 -e 1000000 -a 20 -p 8580 -q",
         "clean": "shx rm -rf dist dist-shakeable",
         "typeorm": "ts-node -r tsconfig-paths/register ./node_modules/.bin/typeorm --config ormconfig-dev.ts",
-        "typeorm:migrate": "yarn typeorm migration:generate -- -n",
+        "typeorm:migrate": "yarn typeorm migration:generate -n",
```

Comment:
This change does not seem to be relevant for this PR. Is it necessary or what does it fix?

Reply:
yarn writes warning about it every time, it says that from version 1.0 double dashes `--` are not necessary

Reply:
I can rollback this change, it only removes yarn warning

Reply:
Ah, ok, cool, keep it then.